### PR TITLE
Back port of PR42 to 1.3.x

### PR DESCRIPTION
**Pull request by jfclere**
Fix for https://issues.redhat.com/browse/MODCLUSTER-761

### CODE REVIEW

File: native/mod_proxy_cluster/mod_proxy_cluster.c [hunk 1]
```diff
@@ -2610,7 +2635,11 @@ static int proxy_cluster_post_config(apr_pool_t *p, apr_pool_t *plog,
             for (j = 0; j < balancer->workers->nelts; j++, workers++) {
                  proxy_worker *worker = *workers;
                  ap_log_error(APLOG_MARK, APLOG_EMERG, 0, s,
+#if MODULE_MAGIC_NUMBER_MAJOR == 20120211 && MODULE_MAGIC_NUMBER_MINOR >= 124
                               "%s BalancerMember are NOT supported %s", balancer->s->name, worker->s->name);
+#else
+                              "%s BalancerMember are NOT supported %s", balancer->s->name, worker->s->name_ex);
+#endif
```
Comment (jajik) [inline native/mod_proxy_cluster/mod_proxy_cluster.c:2642]: I believe there is an error – `name_ex` should be in the `if` branch, not in the `else` one.

File: native/mod_proxy_cluster/mod_proxy_cluster.c [hunk 2]
```diff
@@ -2137,9 +2149,18 @@ static proxy_worker *internal_find_best_byrequests(proxy_balancer *balancer, pro
         apr_table_setn(r->subprocess_env, "BALANCER_CONTEXT_ID", apr_psprintf(r->pool, "%d", (*mynodecontext).context));
         ap_log_error(APLOG_MARK, APLOG_DEBUG, 0, r->server,
                              "proxy: byrequests balancer DONE (%s)",
+#if MODULE_MAGIC_NUMBER_MAJOR == 20120211 && MODULE_MAGIC_NUMBER_MINOR >= 124
+                             mycandidate->s->name_ex
+#else
+
```
Comment (jajik) [inline native/mod_proxy_cluster/mod_proxy_cluster.c:2155]: Unnecessary blank line.

**Review by jajik — COMMENTED** [review]
I find it ok except the two comments below.

File: native/mod_proxy_cluster/mod_proxy_cluster.c (hunk 1 above)
Comment (jfclere) [reply native/mod_proxy_cluster/mod_proxy_cluster.c:2642]: Yes it is, thanks.

**Review by jajik — APPROVED** [review]

**Pull request merged**